Gruntfile.js

```diff
@@ -21,27 +22,24 @@ module.exports = function ( grunt ) {
 				}
 			}
 		},
-		jshint: {
-			options: {
-				jshintrc: true
-			},
-			all: [ '*.js', 'src/**/*.js' ]
-		},
-		jscs: {
+		eslint: {
 			fix: {
 				options: {
 					fix: true
 				},
-				src: '<%= jshint.all %>'
+				src: [
+					'<%= eslint.main %>'
+				]
 			},
-			main: {
-				src: '<%= jshint.all %>'
-			}
+			main: [
+				'*.js',
+				'src/**/*.js'
```

Comment:
It shouldn't be needed, because it's not pulled in by this glob. CI passes locally.